compiler/repl-service/server/src/main/resources/logback.xml

```diff
@@ -9,4 +9,6 @@
     <root level="INFO">
         <appender-ref ref="STDERR"/>
     </root>
+    <logger name="daml.tracelog" level="DEBUG" />
```

Comment:
Is this meant to stay?

Reply:
Yes that is necessary for `trace` messages to be printed out to stdout. We had the same before because the `logback.xml` for `daml-script` was accidentally in the library instead of the binary. I moved it and added a separate `logback.xml` for the repl which disables comments for loading the auth token which only mess with the repl output.

Reply:
:+1: Thanks for clarifying.